cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraTask.java

```diff
@@ -95,6 +90,10 @@ public CassandraTask deserialize(byte[] bytes)
          */
         BACKUP_UPLOAD,
         /**
+         * Task that backup schema for cassandra daemon.
+         */
+        BACKUP_SCHEMA,
```

Comment:
You need to add new types at the end of the Enum or existing tasks will not be deserialized correctly (See https://github.com/mesosphere/dcos-cassandra-service/pull/319 for example).